manager-bundle/composer.json

```diff
@@ -1,12 +1,17 @@
 {
     "name": "contao/manager-bundle",
     "type": "symfony-bundle",
-    "description": "Contao 4 manager bundle",
+    "description": "This bundle contains code necessary to run the Contao 4 Managed Edition.",
```

Comment:
```suggestion
    "description": "Integrates Contao into the Contao managed edition",
```

Reply:
"Managed Edition" is a name to me, and should therefore be uppercase?

Reply:
I don't think so. Then "Contao Core Bundle" would be a name, too, but we are writing it as "Contao core bundle".

Reply:
I think both would be fine for the core bundle. But "Contao Managed Edition" is a name for sure. 
https://duckduckgo.com/?q=contao+managed+edition
https://contao.org/de/news/release-der-contao-managed-edition.html

Reply:
Almost all of the results are in German. Of course it is written "Contao Managed-Edition" there. 😄 

<img width="581" alt="" src="https://user-images.githubusercontent.com/1192057/66030028-c9358780-e500-11e9-9ada-922e1fb9c8a4.png">

This is an English result (and also the official repo) and it clearly says "Contao 4 managed edition":

Reply:
so it should be different in english and german?

Reply:
I have updated my suggestion based on https://github.com/contao/package-metadata/pull/123#issuecomment-537843102